debian/control

```diff
@@ -0,0 +1,17 @@
+Source: gitsh
+Maintainer: thoughtbot <support@thoughtbot.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: debhelper (>= 9), dh-autoreconf, pry, ruby2.2, ruby-bourne,
```

Comment:
There is no `ruby2.2` in `stable`, but there is `ruby2.1`. Is it possible to relax this requirement to "anything compatible with Ruby 2"?